crates/common/tedge_utils/src/file.rs

```diff
@@ -488,7 +488,7 @@ mod tests {
         let file_path = temp_dir.path().join("file").display().to_string();
 
         let user = whoami::username();
-        create_file_with_user_group(&file_path, &user, &user, 0o644, None).unwrap();
+        create_file_with_user_group(&file_path, &user, "root", 0o644, None).unwrap();
```

Comment:
Why not just use the groupname of the user who launched the process?
```suggestion
        let group = uzers::get_current_groupname().unwrap();
        let group = group.to_str().unwrap();
        create_file_with_user_group(&file_path, &user, group, 0o644, None).unwrap();
```

Reply:
I didn't realise we already had a dependency that made getting that information easy, I'll change it